plugins/policy/plugin_impl_policy.go

```diff
@@ -96,20 +102,38 @@ func (p *Plugin) Init() error {
 		},
 	}
 
-	p.aclRenderer = &acl.Renderer{
-		Deps: acl.Deps{
-			Log:        p.Log.NewLogger("-aclRenderer"),
-			LogFactory: p.Log,
-			ContivConf: p.ContivConf,
-			IPv4Net:    p.IPv4Net,
-			UpdateTxnFactory: func() controller.UpdateOperations {
-				p.withChange = true
-				return p.updateTxn
+	if !p.ContivConf.GetIPAMConfig().UseIPv6 {
+
+		p.aclRenderer = &acl.Renderer{
+			Deps: acl.Deps{
+				Log:        p.Log.NewLogger("-aclRenderer"),
+				LogFactory: p.Log,
+				ContivConf: p.ContivConf,
+				IPv4Net:    p.IPv4Net,
+				UpdateTxnFactory: func() controller.UpdateOperations {
+					p.withChange = true
+					return p.updateTxn
+				},
+				ResyncTxnFactory: func() controller.ResyncOperations {
+					return p.resyncTxn
+				},
 			},
-			ResyncTxnFactory: func() controller.ResyncOperations {
-				return p.resyncTxn
+		}
+	} else {
+		p.iptablesRenderer = &iptables.Renderer{
```

Comment:
Perhaps iptablesRenderer could be a configurable alternative for IPv4 as well (?) Might be interesting for performance comparisons, etc.